src/components/PostNote.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Meta, StoryFn } from "@storybook/react";
+import PostNote from "./PostNote";
+
+export default {
+  title: "Components/PostNote",
+  component: PostNote,
+} as Meta;
+
+const Template: StoryFn<typeof PostNote> = (args) => <PostNote {...args} />;
+
+export const Default = Template.bind({});
+Default.args = {
+  title: "University of Tasmania",
+  date: "March 2017 - July 2019",
+  description: "Master of Information Technology and System",
+  drawOutlineTime: 5,
```

Comment:
Prop Explanations
If drawOutlineTime, fadeInTime, or animationDelay have special meaning, consider describing them in the argTypes property or adding a short text block in the story’s description. This helps other developers quickly understand what each prop controls.